proto/proto-backplane-grpc/src/main/proto/deephaven/proto/table.proto

```diff
@@ -267,6 +272,150 @@ message SelectOrUpdateRequest {
   repeated string column_specs = 3;
 }
 
+message Expression {
+  oneof type {
+    string column_name = 1;
+    int64 long_value = 2;
+    string raw = 3;
+  }
+}
+
+message Selectable {
+  string column_name = 1;
+  Expression expression = 2;
+}
+
+message Pair {
```

Comment:
convention elsewhere has been to just encode these as `foo=bar`/`foo` strings - ComboAggregateRequest.Aggregate.match_pairs for example.

i see the point that we can be more descriptive by describing it this way, but for `a=a` we have to write the string twice anyways, and a plain string for each of the two fields is also not accurate, since it can only be java identifiers.

Reply:
I'll add some documentation - but the idea is that `input_column_name` can be empty for the `a=a` case:

```java
    public static Pair adapt(io.deephaven.proto.backplane.grpc.Pair pair) {
        final ColumnName output = ColumnName.of(pair.getOutputColumnName());
        return pair.getInputColumnName().isEmpty() ? output : Pair.of(ColumnName.of(pair.getInputColumnName()), output);
    }
```